torch/functional.py

```diff
@@ -636,6 +636,41 @@ def cartesian_prod(*tensors):
         return handle_torch_function(cartesian_prod, tensors, *tensors)
     return torch._C._VariableFunctions.cartesian_prod(tensors)
 
+def block_diag(*tensors):
+    """Create a block diagonal matrix from provided tensors.
+
+    Arguments:
+        *tensors: One or more tensors with 0, 1, or 2 dimensions. Their scalar
```

Comment:
Minor nit: It looks like the pytorch docstrings aren't consistent about it, but my experience with Python docstrings says `tensors` instead of `*tensors`.

Reply:
The documentation for other functions that use a variable argument list (like `torch.cartesian_prod` and `torch.broadcast_tensors`) have `*tensors` as well.